### ISSUE TITLE
Missing Icons

### ISSUE THREAD

**Issue by ashimokawa** [Freeyourgadget/Gadgetbridge#185]
We need icons for
* language packs (a typical icon would display a/あ)
* firmware files

**Comment by ashimokawa** [comment]
Oh and maybe also something meaning "app" so I do not have to reuse the pebble icon...

@xphnx 
:grinning:

**Comment by xphnx** [comment]
pinged xD

**Comment by ashimokawa** [comment]
@xphnx 
I do not mean to push you...
Now that you got pinged, maybe I should try the monochrome icons you proposed... ;)

**Comment by xphnx** [comment]
Oh, dont worry. You can leave the blue ones.
App icon is for Gadgetbridge? Apps? Exit-to-app? Open-in-app?

**Comment by ashimokawa** [comment]
@xphnx 
Regarding the "App" icon I need something for a generic application type. Opposed to "activity tracker" and "watch face". Right now I use the pebble icon for "generic application", which is confusing because it also shows a watch.

"language pack" and "firmware" would be visible in the fw/app installer, which right now also shows the pebble icon for these types of installable files. 

Last and least important would be a "system app" icon. Right now for debugging purposes I allow to start the build-in "Sports" and "Golf"

To sum up the ultimate with list is (in order of importance)
* generic app
* firmware
* language pack (may or may not be the a/あ style which often represents "translation", don't know how this looks for someone who does not recognize あ, so a better idea is also welcome)
* system app

Would be great If you could do it, but no need to hurry. ;)

**Comment by xphnx** [comment]
What do you think about theese?:

Generic app
![genericapp](https://cloud.githubusercontent.com/assets/4024412/11712967/b6c97df0-9f30-11e5-8db5-b1fcb8184d06.png)

Firmware file 
![firmwarefile](https://cloud.githubusercontent.com/assets/4024412/11713014/08894d96-9f31-11e5-8c00-c63d751fb065.png)

Language file
![languagefile2](https://cloud.githubusercontent.com/assets/4024412/11713037/303b8b10-9f31-11e5-85ef-facf8cde8891.png)
![languagefile1](https://cloud.githubusercontent.com/assets/4024412/11713040/337a5630-9f31-11e5-9b30-60858db9e1de.png)

System app
![systemapp1](https://cloud.githubusercontent.com/assets/4024412/11713053/442a8f4a-9f31-11e5-9078-5670c9f93607.png)
![systemapp2](https://cloud.githubusercontent.com/assets/4024412/11713057/45ebec70-9f31-11e5-806c-95e6eaeb2522.png)

Licence: remix from Google Material Design icons (CC-BY 4.0) https://github.com/google/material-design-icons#license

**Comment by ashimokawa** [comment]
I like the firmware icon!
I like the second language design! (文 is too chinese for me, and あ to japanese)
For system app I like the second one because the first looks like some setup tool and the last on is open (why?) ;)

For the generic app we then would have to choose the first one for consistency and maybe it would make sense to also replace the watchface icon to match that design?

**Comment by ashimokawa** [comment]
@xphnx 
do you also have them as svg files, I can just convert them to png with my script for all sizes.

**Comment by xphnx** [comment]
Lets give another chance to the second design xD... What about theese for the apps icons (generic/system/watchface)? Is something like this what you're looking for watchface icon? Of course, I will attach the svg files.

Design 1 
![genericapp1](https://cloud.githubusercontent.com/assets/4024412/11742974/54728af0-a004-11e5-9f97-9a3442f97d07.png)
![systemapp1](https://cloud.githubusercontent.com/assets/4024412/11742975/56e550ba-a004-11e5-959f-cc602c7e3af3.png)
![watchapp1](https://cloud.githubusercontent.com/assets/4024412/11742977/58e9e7d6-a004-11e5-995c-9e9cd04a815e.png)

Design 2
![genericapp2](https://cloud.githubusercontent.com/assets/4024412/11743102/1e0ef646-a005-11e5-9c8c-1c8a565dec4e.png)
![systemapp2](https://cloud.githubusercontent.com/assets/4024412/11742984/622a054c-a004-11e5-8b72-4cc986846594.png)
![watchapp2](https://cloud.githubusercontent.com/assets/4024412/11742986/63a09904-a004-11e5-93e7-fce32a0f8d6d.png)

**Comment by xphnx** [comment]
svgs here: https://my.owndrive.com/index.php/s/T3PFgxE74g4ISJQ

**Comment by ashimokawa** [comment]
Thanks, I think I will use Design 2. ;)

**Comment by ashimokawa** [comment]
@xphnx 
I checked these in.
Now I forgot the ic_activitytracker.png file, it does not fit now that new design, can you do that also? :sweat_smile:

**Comment by xphnx** [comment]
Activitytracker1 (maybe better for beatrate?)
![ic_activitytracker1](https://cloud.githubusercontent.com/assets/4024412/11806306/6a0af15c-a312-11e5-9f01-e0a9767661ae.png)

Activitytracker2
![ic_activitytracker2](https://cloud.githubusercontent.com/assets/4024412/11806309/6ffba944-a312-11e5-8540-85259e902b16.png)

Activitytracker3
![ic_activitytracker3](https://cloud.githubusercontent.com/assets/4024412/11806311/7519d0a4-a312-11e5-9f39-3fe77741e61e.png)

Don't you think that putting a frame with its icon is a little redundant?, and perhaps would be better to only show the icon of each one (dispensign with the frame)? (that would return us to the previous icons, I guess)

svgs here: https://my.owndrive.com/index.php/s/4WY1gaRGy2PTTuM

**Comment by ashimokawa** [comment]
I guess you are right, I also noticed that it makes the actual iconography really tiny.
Maybe we should revert, keeping firmware file and language file icons...
We would then need  "application" and "system application" (a.k.a build-in application) with the "old" design.

**Comment by xphnx** [comment]
Another batch:
Watchapp
![ic_watchapp](https://cloud.githubusercontent.com/assets/4024412/11841804/00737d88-a400-11e5-9815-00d4034dce95.png)

Systemapp
![ic_systemapp](https://cloud.githubusercontent.com/assets/4024412/11841810/083cabe8-a400-11e5-8f94-24f1e8f4d24a.png)

Activitytracker
![ic_activitytracker1](https://cloud.githubusercontent.com/assets/4024412/11841825/176d24f8-a400-11e5-9597-555088a2750c.png)
![ic_activitytracker2](https://cloud.githubusercontent.com/assets/4024412/11841823/15718c66-a400-11e5-8c99-36bd76216240.png)

These designs would be consistent with the iconography also used by pebble in their website
![websitepebble](https://cloud.githubusercontent.com/assets/4024412/11841909/883ed3e8-a400-11e5-8d84-b4e38f53e432.png)

svgs: https://my.owndrive.com/index.php/s/5mlVnBtlWo3l604

**Comment by ashimokawa** [comment]
Nice, I will use these for 0.7.0 (next release).

For activity the bars seem more appropriate since basically it could be anything that collects data in the background (battery+ for example).

Many thanks!

**Comment by xphnx** [comment]
You are welcome! ;)

**Comment by ashimokawa** [comment]
0.7.0 and 0.7.1 are out - with the new icons. Closing. ;)
